css/styles.css

```diff
@@ -51,20 +62,26 @@ button {
   cursor: pointer;
   border-radius: 5px;
   display: block;
-  margin: 20px auto 0;
-  margin-right: 50px;
 }
 
 button:hover {
   background-color: #6a1b9a;
 }
 
-img#photo-preview {
-  display: block;
+.img-container {
+  width: 100%;
+  max-width: 300px;
+  height: 200px;
+  display: flex;
```

Comment:
You don't need the max-width here. It's actually forcing your image to be smaller than what you want. I would just leave this with a `width: 100%` and remove the `max-width` and `height`